### PR TITLE
Updated all acceptance tests but one to start strong

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: https://central.tri.be/issues/42077

This PR fixes the currently failing or broken existing TEC acceptance tests. Of note:

* division of testing phases: all the modified tests use the *arrange, act, assert* structure to avoid assertions where those do not belong
* much set up is done using WordPress methods using the WPBootrstrapper module
* the tests should now be agnostic of the initial db configuration

### CODE REVIEW

File: tests/acceptance/CreateATagCept.php [hunk 1]
```diff
@@ -1,17 +1,32 @@
...
-$I = new AcceptanceTester( $scenario );
 
-//Activate TEC Calendar
-$I->am( 'administrator' );
-$I->wantTo( "verify that a tag can be created" );
+// @group: settings
+$I = new AcceptanceTester($scenario);
+
+$I->am('administrator');
+$I->wantTo("verify that a category can be created");
```
Comment (borkweb) [inline tests/acceptance/CreateATagCept.php]: Copy/paste error here, I think. Should be `verify that a tag can be created`

**Comment by borkweb** [comment]
:+1:

**Comment by lucatume** [comment]
I will not merge and delete straight to have @rmacdon465 review it too and have a place to discuss changes

**Comment by lucatume** [comment]
putting this back into code review as there have been some changes worth noting:

* removed the `phantomjs` suite as we are using webdrivers to do acceptance already
* refactored the CSV import test

**Pull request merged**